### PR TITLE
Proxito: test new implementation more broadly

### PR DESCRIPTION
**Pull request by stsewd**
With this we can now exclude some projects from using the new implementation while testing it more broadly
by changing the date of the feature.

When we are ready to enable it for all projects, we can remove the USE_UNRESOLVER_WITH_PROXITO, and leave the other setting while we migrate the missing projects.

To avoid introducing another DB call, I'm caching the features from a project.

### CODE REVIEW

File: readthedocs/projects/models.py [hunk 1]
```diff
@@ -2015,6 +2015,10 @@ def add_features(sender, **kwargs):
                 "Proxito: Use new unresolver implementation for serving documentation files."
             ),
         ),
+        (
+            USE_OLD_PROXITO_IMPLEMENTATION,
+            _("Proxito: Use old implementation for serving documentation files."),
```
Comment (humitos) [inline readthedocs/projects/models.py:2020]:
```suggestion
            _("Proxito: Use old Proxito implementation (no unresolver) for serving documentation files."),
```

Make this sentence more explicit here.

**Review by humitos — COMMENTED** [review]
Looks good -- I'm a little confused about what will happen if a project has both feature flags (old and new implementation) enabled at the same time?

**Comment by stsewd** [comment]
>  I'm a little confused about what will happen if a project has both feature flags (old and new implementation) enabled at the same time?

The USE_OLD_PROXITO_IMPLEMENTATION feature takes precedence.

**Comment by humitos** [comment]
> The USE_OLD_PROXITO_IMPLEMENTATION feature takes precedence.

👍🏼 -- can you add a comment about this in the Feature model (or even in the description of it) so we are all aware when enabling/touching this?

**Review by humitos — APPROVED** [review]

**Comment by humitos** [comment]
I'm merging this PR so it goes out today because we want to move forward here and test this more.

@stsewd please, if you have the time, add the comment/description in another PR.

**Pull request merged**